README.md

```diff
@@ -1,60 +1,124 @@
+![Screenshot](.github/screenshot.png?raw=true)
 
-### Docker
+## Requirements
 
-```bash
-docker build -t srcd/code-annotation .
-docker run --env-file .env --rm -p 8080:8080 srcd/code-annotation
-```
+### Global dependencies
 
-### Non-docker
+You should already have [Go installed](https://golang.org/doc/install#install), and properly [configured the $GOPATH](https://github.com/golang/go/wiki/SettingGOPATH)
 
-```bash
-go get github.com/src-d/code-annotation/...
-cd $GOPATH/github.com/src-d/code-annotation
-make serve
+```
+go version; # prints your go version
+echo $GOPATH; # prints your $GOPATH path
 ```
 
-## Development
+The project must be under $GOPATH, as required by the Go tooling.
+You should be able to navigate into the source code by running:
 
-Backend:
+```
+cd $GOPATH/src/github.com/src-d/code-annotation
+```
+
+You need also [Yarn v1.x.x installed](https://yarnpkg.com/en/docs/install)
 
 ```
-make gorun
+yarn --version; # prints your Yarn version
 ```
 
-Frontend:
+### Github OAuth tokens
+
+1. You need an OAuth application on GitHub. See [how to create OAuth applications on GitHub](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/).
+
+In order to be able to use this application while running the tool locally, make sure you add http://127.0.0.1:8080/oauth-callback to the authorization callback URL field.
+
+2. Copy `.env.tpl` to `.env`.
+
+3. Retrieve the values for your application's Client ID and Client Secret from the [GitHub Developer Settings page](https://github.com/settings/developers) and add them to the end of the corresponding lines in .env.
+
+## Installation
 
-If you want to benifit from frontend hot reloading feature this line in your `.env` file:
+You need to satisfy all [project requirements](#requirements), and then run:
 
+```bash
+$ go get github.com/src-d/code-annotation/...
+$ cd $GOPATH/github.com/src-d/code-annotation
+$ make serve
 ```
-UI_DOMAIN=http://127.0.0.1:3000
+
+This will start a server locally, which you can access on [http://localhost:8080](http://localhost:8080)
+
+## Importing and Exporting Data
+
+### Import File Pairs for Annotation
+
+The file pairs must be provided via an [SQLite](https://sqlite.org/) database. The database **must follow the expected schema**, please [follow this link](./cli/examples/import/example.sql) to see an example.
+
+The `import` command will use those file pairs to create a new [SQLite](https://sqlite.org/) or [PostgreSQL](https://www.postgresql.org/) database that will be used internally by the Annotation Tool. The destination database does not need to be empty, new imported file pairs can be added to previous imports.
+
+_Note_: duplicate entries are not filtered, so running an import multiple times will result in repeated rows.
+
+To use it, run it as:
+
+```bash
+$ import <path-to-sqlite.db> <destination-DSN>
 ```
 
-And then restart server.
+Where the `DSN` (Data Source Name) argument must be one of:
```

Comment:
In the export section, it was added a hint about the command parameters:
>_In this case, origin will be the internal database, and destination the new database._

I think it would be a good idea to do so here:
> _In this case, `<path-to-sqlite.db>` is the database with the pairs to import, and `<destination-DSN>` will be the DSN of the **internal database**._